classes/PodsAdmin.php

```diff
@@ -2441,4 +2444,197 @@ function configuration( $pod = null, $full_field_info = false ){
 
 	}
 
+    /**
+     * Build UI for extending REST API, if makes sense to do so.
+     *
+     * @todo maybe disable if default routes don't exist since WP < 4.5 and plugin isn't installed?
+     *
+     * @since 2.5.6
+     *
+     * @access protected
+     */
+    protected function rest_admin() {
+        if( function_exists( 'register_rest_field' ) ) {
+            add_filter( 'pods_admin_setup_edit_field_options', array( $this, 'add_rest_fields_to_field_editor' ), 12, 2 );
+            add_filter( 'pods_admin_setup_edit_options', array( $this, 'add_rest_settings_tab_fields' ), 12, 2 );
+            add_filter( 'pods_admin_setup_edit_tabs', array( $this, 'add_rest_settings_tab' ), 12, 2 );
+            add_filter( 'pods_admin_setup_edit_field_tabs', array( $this, 'add_rest_field_tab' ), 12 );
+        }
+
+    }
+
+    /**
+     * Check if Pod type <em>could</em> extend core REST API response
+     *
+     * @since 2.5.6
+     *
+     * @access protected
+     *
+     * @param array $pod
+     *
+     * @return bool
+     */
+    protected function restable_pod( $pod ) {
+        $type =  $pod[ 'type' ];
+        if( in_array( $type, array(
+                'post_type',
+                'user',
+                'taxonomy'
+            )
+        )
+        ) {
+            return true;
```

Comment:
Not sure I follow you. This check is to see if the Pod could extend a core route. Unrelated to custom routes...